packages/react/src/patterns/sub-patterns/PictogramItem/PictogramItem.js

```diff
@@ -47,7 +47,7 @@ const PictogramItem = ({
         <ContentItem
           heading={heading}
           copy={copy}
-          cta={{ style: 'text', ...cta }}
+          cta={cta != null ? { style: 'text', ...cta } : null}
```

Comment:
does it work if we do `{cta && {style: 'text', ...cta}}`

Reply:
yes, it does, I've updated it on the latest version